diagnostic.md

```diff
@@ -71,13 +72,28 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+relative, b/c it is going to take us to the 'planing' folder inside the 'project'
+folder.  You might have multiple sub-folders called 'planing' and using 'cd planing'
+will take you to the 'planing' folder relative to your current parent directory.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+I would want to use a absolute path b/c you could eaisly end up with a folder structer
```

Comment:
In this case I would use a relative path since the image file lives inside our current working directory, project. As a result, an absolute path is not needed to access it.